helm/cluster-cloud-director/values.yaml

```diff
@@ -103,6 +103,12 @@ userContext:
   secretRef:
     secretName: ""  # Name of the pre-existing secret containing the credentials of the VCD user.
 
+pod:
```

Comment:
Will this ever need to be changed? i.e. do we need to expose it?

Reply:
We usually expose it (e.g. https://github.com/giantswarm/cert-manager-app/blob/1b995e85933d85ff739ff9841d71c1c875986aea/helm/cert-manager-app/values.yaml#L250) but I'm fine with hiding it.

Reply:
Since the interface is exposed to the customer I'd lean towards hiding it to avoid cluttering the values. @erkanerol opinion?

Reply:
As far as I understand, we should hide these kind of configuration under `.internal` section. See https://github.com/giantswarm/roadmap/issues/1733

On the other hand, we don't follow the policy for other parameters here :D 

At least, we should rename it with a better name like `helmHookPod`.

Reply:
It would be great if the properties could already be placed in the intended root level structure. Otherwise this would have to change soon.

Reply:
Moved it _helpers.tpl https://github.com/giantswarm/cluster-cloud-director/pull/78/commits/89c46e5e4cd03e64f1581f08f7c562dd59780903